src/obb_demo/processor.cljs

```diff
@@ -62,7 +62,7 @@
     #_(println "game" (simplifier/clean-result {:board game}))
     (println "--" player actions)
     (if (= :final (game/state game))
-      {:game (deployed-game)}
+      {:game (deployed-game {:mode (get game-data :mode :annihilation)})}
```

Comment:
Did you have to set the default game mode? Supposedly, my changes to `game-progress/new-game` defined default values for the options, and that value had `:annihilation` as the default for the game mode. If that is not the case, it is a bug.

Generally speaking, I think we should try to define the defaults in the code that handles game creation.